pkg/systemd/services.html

```diff
@@ -139,6 +139,25 @@
           </table>
         </div>
         {{/NotMetConditions.length}}
+        {{#Relationships}}
+        {{#Units.length}}
+        <div class="list-group-item">
+          <table>
+            <tr>
+              <td>
+                <span>{{Name}}</span>
+                <ul>
+                {{#Units}}
+                  <li><a href='#/{{.}}'>{{.}}</a></li>
```

Comment:
Does that work with template units (foo@123.service) and units with "funny" characters in the name? I. e. does that need escaping, or does mustache magically do that for you?

Reply:
it says that it does HTML escaping when you use {{ }}.  I don't know if that includes escaping appropriate for inside of attribute values.  Good call: i'll check.

Reply:
Make sure to add a funny unit name to the corresponding tests :)

Reply:
The escaping works properly.  I added this:
{ Name: _("b0rked"),          Units: ["&amp;", "'", '"', "</xyz>"] },
and got this:

b0rked
* &amp;
* '
* "
* </xyz>